### PR TITLE
Default CI

### PR DESCRIPTION
**Pull request by kkalinowski-reef**
- running `nox -s test` during pull requests to the master/main branch
- running `nox -s lint` and/or spellchecker as an option
- generated code that is passing flake linting without issues (with and without monitoring, monitoring view and celery) (various manual formatting fixes were applied)
- typos discovered by the codespell were fixed
- before tests, CI sets up everything exactly as the readme describes (`freezegun` library from `test_example` uses the DB and thus ensures that this part works as well, `test_hello_world` doesn't use the DB and just ensures that startup was successful)

### CODE REVIEW

File: {{cookiecutter.repostory_name}}/app/src/{{cookiecutter.django_project_name}}/settings.py [hunk 1]
```diff
@@ -2,7 +2,9 @@
 Django settings for {{cookiecutter.django_project_name}} project.
 """
 
+{% if cookiecutter.use_celery == "y" %}
 from datetime import timedelta
+{% endif %}
```
Comment (vbaltrusaitis-reef) [inline {{cookiecutter.repostory_name}}/app/src/{{cookiecutter.django_project_name}}/settings.py:7]: You judiciously added dashes to remove unneeded empty lines everywhere, which is very much appreciated. Why not here? Did the linter caught all the other ones?

File: {{cookiecutter.repostory_name}}/.github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        uses: actions/setup-python@v4
+        with:
+          python-version: {% raw %}${{ env.PYTHON_DEFAULT_VERSION }}{% endraw %}
+          cache: "pip"
+      - name: Prepare environment
+        run: ./setup-dev.sh
+      - name: Install dependencies
+        run: python -m pip install --upgrade nox pip setuptools
+      - name: Run dockerized services
+        run: docker-compose up -d
```
Comment (vbaltrusaitis-reef) [inline {{cookiecutter.repostory_name}}/.github/workflows/ci.yml:57]: I assume that's pre-installed in the `ubuntu-latest`, right?

**Review by vbaltrusaitis-reef — APPROVED** [review]

File: {{cookiecutter.repostory_name}}/.github/workflows/ci.yml (hunk 2 above)
Comment (kkalinowski-reef) [reply {{cookiecutter.repostory_name}}/.github/workflows/ci.yml:57]: https://github.com/kkalinowski-reef/test-single-test-ci/actions/runs/3891510984/jobs/6641822087

It seems so. Worked without issues.

File: {{cookiecutter.repostory_name}}/app/src/{{cookiecutter.django_project_name}}/settings.py (hunk 1 above)
Comment (kkalinowski-reef) [reply {{cookiecutter.repostory_name}}/app/src/{{cookiecutter.django_project_name}}/settings.py:7]: I worked on these lines just to ensure that the linter is not complaining. Completely forgot about this. Thank you!

**Pull request merged**